### ISSUE TITLE
Color picker `INPUT` is too short

### ISSUE THREAD

**Issue by hideheader** [JasonBarnabe/userstyles#83]
The `INPUT` permits 7 characters ('#234567'), but the longest color name is 20 characters ('lightgoldenrodyellow').

**Comment by JasonBarnabe** [comment]
It's designed to accept only RGB values. This is partially because the color picker only considers RBG to be correct. #44

**Comment by hideheader** [comment]
No. In fact, every color is in the caboodle except `rebeccapurple`, which we can do without.
https://github.com/JasonBarnabe/userstyles/blob/master/public/javascripts/jscolor.js#L512-L514

**Comment by JasonBarnabe** [comment]
Yes. In fact, that's the code I added in #44 that only added partial support.

**Comment by hideheader** [comment]
And it's great code, really it is, but unless `maxlength` is increased from 7 to 20 on `INPUT.color` then it isn't possible to type in the color names.
